src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,88 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import racingcar.utils.Validator;
+
+public class Cars {
+    private final List<Car> cars;
+
+    private Cars(List<Car> cars) {
+        this.cars = new ArrayList<>(cars);
+        validate();
+    }
+
+    public static Cars from(List<Car> cars) {
+        return new Cars(cars);
+    }
+
+    private void validate() {
+        Validator.containDuplicate(fetchCarNames());
+    }
+
+    public void allMove() {
+        for (Car car : cars) {
+            car.move(new RandomMovementStrategy());
+        }
+    }
+
+    public List<String> findRaceWinners() {
+        int maxDistance = findMaxDistance();
+        return findWinners(maxDistance);
+    }
+
+    private List<String> findWinners(int maxDistance) {
+        List<String> winners = new ArrayList<>();
+        for (Car car : cars) {
+            if (maxDistance == car.getDistance()) {
+                winners.add(car.getName());
+            }
+        }
+        return winners;
+    }
```

Comment:
findMaxDistance()에서도 for문이 한번 도는 구조라 for문이 두번 돌게 되네용.
for문 한번만으로도 원하는 결과를 만들 수 있을 것 같아용

```suggestion
public List<String> findRaceWinners() {
    int maxDistance = 0;
    List<String> winnerCarNames = new ArrayList<>();
    for(Car car : cars) {
        int carDistance = car.getDistance()
        String carName = car.getName()

        if(carDistance == maxDistance) {
            winnerCarNames.add(carName);
            continue;
        }
        if(carDistance > maxDistance) {
            maxDistance = carDistance
            winnerCarNames.clear()
            winnerCarNames.add(carName)
        }
    }
}
```